src/main/java/com/danielagapov/spawn/Controllers/TestController.java

```diff
@@ -46,4 +58,43 @@ public void setName(String name) {
     public void setAge(Integer age) {
         this.age = age;
     }
-}
\ No newline at end of file
+}
+
+class User {
+    private String username;
+    private String firstName;
+    private String lastName;
+    private String bio;
+    
+    public String getUsername() {
+        return this.username;
+    }
+
+    public String getfirstName() {
```

Comment:
```suggestion
    public String getFirstName() {
```
Suggestion example: Use proper camel case